uwuno/src/main/java/com/learning/uwuno/room.java

```diff
@@ -33,12 +45,41 @@ public void setRoomName(String roomName) {
         this.roomName = roomName;
     }
 
+    public void setRoomStatus(Status status) {
+        this.roomStatus = status;
+    }
+
+    public Status getRoomStatus() {
+        return roomStatus;
+    }
+
     public String getUid() {
         return uid;
     }
 
+    public deck getDeck() {
```

Comment:
Is there a reason we are returning the deck? Originally implemented this without so noone can accidentally edit the deck since Java only passes by reference. A quick search I can't quite see what this is used for (Unless its for Spring's JSON -> POJO contructing? If that's the case maybe have to just call constructor manually after parsing like some of the other commands?)